data/sql/updates/pending_db_characters/rev_1642821969869259834.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_characters` (`sql_rev`) VALUES ('1642821969869259834');
+
+-- column `type` contains type of logged action
+ALTER TABLE `log_money` ADD COLUMN `type` char(255) NOT NULL AFTER `date`;
```

Comment:
Hey, thanks for your PR, it's much appreciated. However, the design here could be improved as I do believe a tinyint field would be better than a char one. It serves the same purpose but there's less data stored overall.

0 - mail
1 - trade
...etc

Reply:
You mean that I must to replace the words with numbers here and at the core?
type 1 = mail, type 2 =trade ... ?
Then we will have to describe it somewhere in the wiki...

Reply:
Fixed and checked in game all 6 types.